css/properties/border-top-width.json

```diff
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "border-top-width": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-top-width",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
```

Comment:
This should be "android_webview". I need to update the versions we've collected to talk about the weird android versions before android started to use chromium versions.